variables.tf

```diff
@@ -47,6 +47,11 @@ variable "versioning" {
   default     = {}
 }
 
+variable "encryption" {
```

Comment:
I think we should have a more descriptive name.

```suggestion
variable "encryption_key_name" {
```

Reply:
Yeah I started with a longer name but I ended up with a short one. I'll get these sorted and pushed soon :)

Reply:
I've gone with `encryption_key_names` plural just because it's a list but I can amend the commit to `encryption_key_name` if you'd prefer

Reply:
Either sounds good!